### ISSUE TITLE
Install fails

### ISSUE THREAD

**Issue by marc-vdm** [Pan6ora/ab-plugin-Notebook#2]
I created an AB enviornment specifically to test this plugin install with:  `conda create -n ab-plugin activity-browser` This installed AB `2.8.0`, I then `conda activate ab-plugin` and `conda install -c pan6ora -c conda-forge ab-plugin-notebook`, this took some time to solve, but it worked in the end.

I then opened AB, created a new project (`ei39`) and enabled the plugin from the popup, after I clicked `Finish`, I got the following:
```
Loaded project: ei39
Traceback (most recent call last):
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\site-packages\activity_browser\controllers\plugin.py", line 70, in add_plugin
    plugin.load()
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\site-packages\ab_plugin_notebook\__init__.py", line 26, in load
    self.server = subprocess.Popen([
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

 
Plugin-Notebook: server not running 
```

When I re-open AB and this project, I get the following:
```
Error: plugin ab_plugin_notebook not installed
Reset metadata.
Loaded project: ei39
```
The plugin popup does say the plugin is enabled though

**Comment by marc-vdm** [comment]
The following output also happened during the plugin install

```
The following packages will be SUPERSEDED by a higher-priority channel:

  activity-browser   conda-forge::activity-browser-2.8.0-p~ --> pan6ora::activity-browser-2.7.5+pm2.1.0-py_0

The following packages will be DOWNGRADED:

  gst-plugins-base                        1.22.3-h001b923_1 --> 1.21.3-h001b923_1
  gstreamer                               1.22.3-h6b5321d_1 --> 1.21.3-h6b5321d_1
  icu                                       72.1-h63175ca_0 --> 70.1-h0e60522_0
  lcms2                                     2.15-h3e3b177_1 --> 2.12-h2a16943_0
  libclang                        16.0.3-default_h8b4101f_1 --> 15.0.7-default_h77d9078_2
  libclang13                      16.0.3-default_h45d3cf4_1 --> 15.0.7-default_h77d9078_2
  libjpeg-turbo                          2.1.5.1-hcfcfb64_0 --> 2.1.4-hcfcfb64_0
  libtiff                                  4.5.1-h6c8260b_0 --> 4.2.0-h0c97f57_3
  openjpeg                                 2.5.0-ha2aaf27_2 --> 2.4.0-hb211442_1
  pillow                               9.5.0-py39ha9166d5_1 --> 8.2.0-py39h1a9d4f7_1
  pyside2                             5.15.8-py39h56cbfbc_2 --> 5.15.5-py39h1d2b913_1
  qt-main                                5.15.8-h2c8576c_11 --> 5.15.6-h9580fe5_6
  qt-webengine                            5.15.8-h5b1ea0b_0 --> 5.15.4-h0bdee70_3
```

I have `strict` channel priority set in conda because of other things I'm working on that would otherwise break. However, I'm not sure we want to overwrite AB from your channel if we can avoid it.

**Comment by Zoophobus** [comment]
In the requirements maybe it should read `activity-browser >= 2.8`?

**Comment by Pan6ora** [comment]
Hi @marc-vdm ,

> The following output also happened during the plugin install
> 
> ```
> The following packages will be SUPERSEDED by a higher-priority channel:

This is because at the time I was creating plugins I had to maintain my own Activity Browser version on Anaconda. Now that the plugin system has been merged it is not necessary anymore.
I just removed it so this message should not happen anymore.

> In the requirements maybe it should read `activity-browser >= 2.8`?

I also changed that in every existing plugins.

**Comment by Pan6ora** [comment]
@marc-vdm about your first message, on which operating system are you ?

**Comment by marc-vdm** [comment]
Thanks for the quick reaction! I re-created the environment and re-installed the plugin `0.2.6` (same instructions as above) and this is the error I get when I enable the plugin now:

```
(ab-plugin) C:\>activity-browser
-hiding tab: History

DevTools listening on ws://127.0.0.1:3999/devtools/browser/49896661-1020-49bf-8bb8-1953b945a5ae
-hiding tab: Activity Details
-hiding tab: Characterization Factors
-hiding tab: Graph Explorer
-hiding tab: LCA results
Loading plugin ab_plugin_notebook
Loading user settings:
dirpath already loaded
Reset project settings directory to: C:\Users\meidemtvander\AppData\Local\pylca\Brightway3\default.c21f969b5f03d33d43e04f8f136e7682
Loaded project: default
Brightway2 data directory: C:\Users\meidemtvander\AppData\Local\pylca\Brightway3
Brightway2 active project: default
Qt Version: 5.15.8
Remote debugging server started successfully. Try pointing a Chromium-based browser to http://127.0.0.1:3999
Traceback (most recent call last):
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\site-packages\activity_browser\controllers\plugin.py", line 70, in add_plugin
    plugin.load()
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\site-packages\ab_plugin_notebook\__init__.py", line 26, in load
    self.server = subprocess.Popen([
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

Traceback (most recent call last):
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\site-packages\activity_browser\ui\tables\plugins.py", line 51, in mousePressEvent
    self.model.sync()
  File "C:\Users\meidemtvander\.conda\envs\ab-plugin\lib\site-packages\activity_browser\ui\tables\models\plugins.py", line 36, in sync
    self.updated.emit()
RuntimeError: Internal C++ object (PluginsModel) already deleted.
```
AB then hangs and doesn't close when I click the close button, I need to `ALT`+`F4`

**Comment by Pan6ora** [comment]
The problem should be fixed in 0.2.7 (tested on windows 11 with Anaconda).

**Comment by marc-vdm** [comment]
Thanks! This is resolved.
